File: test/test-3_16-08-24/Que19.c
/*
19] Write a C program to find the length of a string without using any built-in functions.
*/

#include <stdio.h>
#include <string.h>

int length(char str[20])
{
    int i = 0;
    while(str[i] != '\0')
    {
        i++;
    }
    return i;
}

int main()
{
    char str[20];
    printf("Enter string\n");
    scanf("%s",str);
    printf("The length of string is: %d",length(str));
    return 0;
}

File: tops-work/PYTHON/Assignment/Fundamentals-of-Python/que19.py
"""
Que19. Write a Python program to find the first appearance of the substring
'not' and 'poor' from a given string, if 'not' follows the 'poor', replace the
whole 'not'...'poor' substring with 'good'. Return the resulting string.
"""

name = input("Enter any string: ")
sub_str1 = "not"
sub_str2 = "poor"
print("first appearance of 'not'",name.index("not"))
print("first appearance of 'poor'",name.index("poor"))
if (name.index("poor") - name.index("not")) == 4:
    new_name = name[0:name.index("not")] + "good" + name[(name.index("poor") + 4):]
    print(new_name)
else:
    print(name)